Description: Remove documentation, which was shipped as binary
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2015-08-27

--- a/src/Doc/CMakeLists.txt
+++ b/src/Doc/CMakeLists.txt
@@ -5,28 +5,28 @@
 
 
 if(BUILD_GUI)
-    FILE(GLOB QCH_FILES freecad.qch.part*)
-    list(SORT QCH_FILES)
-    set (DEST_FILE ${CMAKE_BINARY_DIR}/doc/freecad.qch)
-    foreach(SRC_FILE ${QCH_FILES})
-        if(${SRC_FILE} IS_NEWER_THAN ${DEST_FILE})
-            EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/src/Tools/catfiles.py
-                  --outputfile=${DEST_FILE} ${QCH_FILES}
-                  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-                  OUTPUT_STRIP_TRAILING_WHITESPACE)
-            break()
-        endif()
-    endforeach()
+    #FILE(GLOB QCH_FILES freecad.qch.part*)
+    #list(SORT QCH_FILES)
+    #set (DEST_FILE ${CMAKE_BINARY_DIR}/doc/freecad.qch)
+    #foreach(SRC_FILE ${QCH_FILES})
+        #if(${SRC_FILE} IS_NEWER_THAN ${DEST_FILE})
+            #EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/src/Tools/catfiles.py
+                  #--outputfile=${DEST_FILE} ${QCH_FILES}
+                  #WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+                  #OUTPUT_STRIP_TRAILING_WHITESPACE)
+            #break()
+        #endif()
+    #endforeach()
 
-    configure_file(freecad.qhc ${CMAKE_BINARY_DIR}/doc/freecad.qhc COPYONLY)
-    INSTALL(FILES
-        ${CMAKE_BINARY_DIR}/doc/freecad.qch
-        DESTINATION ${CMAKE_INSTALL_DOCDIR}
-    )
+    #configure_file(freecad.qhc ${CMAKE_BINARY_DIR}/doc/freecad.qhc COPYONLY)
+    #INSTALL(FILES
+        #${CMAKE_BINARY_DIR}/doc/freecad.qch
+        #DESTINATION ${CMAKE_INSTALL_DOCDIR}
+    #)
 endif(BUILD_GUI)
 
 INSTALL(FILES
-    freecad.qhc
+    #freecad.qhc
     ThirdPartyLibraries.html
     DESTINATION ${CMAKE_INSTALL_DOCDIR}
 )
